### PR TITLE
dev-python/pygdbmi: new package, add 0.11.0.0

### PR DESCRIPTION
**Pull request by parona-source**
For drkonqi:6

### CODE REVIEW

File: dev-python/pygdbmi/pygdbmi-0.11.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+"
+# no tests in sdist
+SRC_URI="https://github.com/cs01/pygdbmi/archive/refs/tags/v${PV}.tar.gz -> ${P}.gh.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND=">=sys-devel/gdb-7.6"
+BDEPEND="test? ( ${RDEPEND} )"
```
Comment (mgorny) [inline dev-python/pygdbmi/pygdbmi-0.11.0.0.ebuild:25]: det already does that for you.

File: dev-python/pygdbmi/pygdbmi-0.11.0.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+inherit distutils-r1
+
+DESCRIPTION="A library to parse gdb mi output and interact with gdb subprocesses"
+HOMEPAGE="
+	https://cs01.github.io/pygdbmi/
+	https://github.com/cs01/pygdbmi
+	https://pypi.org/project/pygdbmi/
+"
+# no tests in sdist
+SRC_URI="https://github.com/cs01/pygdbmi/archive/refs/tags/v${PV}.tar.gz -> ${P}.gh.tar.gz"
```
Comment (mgorny) [inline dev-python/pygdbmi/pygdbmi-0.11.0.0.ebuild:18]: Please explode and wrap lines, see other python@ ebuilds.

File: dev-python/pygdbmi/pygdbmi-0.11.0.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{11..12} )
+
+inherit distutils-r1
+
+DESCRIPTION="A library to parse gdb mi output and interact with gdb subprocesses"
+HOMEPAGE="
+	https://cs01.github.io/pygdbmi/
+	https://github.com/cs01/pygdbmi
```
Comment (mgorny) [inline dev-python/pygdbmi/pygdbmi-0.11.0.0.ebuild:14]: Add a trailing slash for consistency.

File: dev-python/pygdbmi/pygdbmi-0.11.0.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+	https://pypi.org/project/pygdbmi/
+"
+# no tests in sdist
+SRC_URI="https://github.com/cs01/pygdbmi/archive/refs/tags/v${PV}.tar.gz -> ${P}.gh.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND=">=sys-devel/gdb-7.6"
```
Comment (mgorny) [inline dev-python/pygdbmi/pygdbmi-0.11.0.0.ebuild:24]: Explode to make appending easier.

**Review by mgorny — CHANGES_REQUESTED** [review]

File: dev-python/pygdbmi/pygdbmi-0.11.0.0.ebuild (hunk 1 above)
Comment (parona-source) [reply dev-python/pygdbmi/pygdbmi-0.11.0.0.ebuild]: Didn't know about this.

**Comment by parona-source** [comment]
Bumped gdb version because despite what upstream claimed about 7.3 and up I noticed them using a feature available since 9.6. 

The feature being the mi3 interface.

**Pull request closed without merging**